src/autodiscovery.ts

```diff
@@ -495,8 +416,9 @@ export class AutoDiscovery {
      * @return {Promise<object>} Resolves to the returned state.
      * @private
      */
-    static async _fetchWellKnownObject(url) {
+    private static async fetchWellKnownObject(url: string): Promise<IWellKnownConfig> { // TODO: TYPES
```

Comment:
what's the todo here for, looks typed to me?

Reply:
Thanks for the heads up! Fixed now